### DISCUSSION TITLE
[Bug] Missing output (stderr) from action run

### DISCUSSION THREAD

**Discussion by Flamefire** [community/community#26949] [Actions]
<p>I’m running a Docker Github action which in this case results in an error an appropriate message being printed.</p>
<p>However this important message is missing from the parsed log.</p>
<p>Example: <a href="https://github.com/Flamefire/s25client/runs/682872393?check_suite_focus=true" rel="nofollow noopener">https://github.com/Flamefire/s25client/runs/682872393?check_suite_focus=true</a></p>
<p>Expand the “Formatting” node and see that the first printed message starts with “usage: run-clang-format.py” and no other output is shown</p>
<p>Now open the raw log (<a href="https://pipelines.actions.githubusercontent.com/rJZXGyMx0DPiJ1uuZOVUzJH5mFZ5flbMm8uZtBjQquT4PzUNKW/_apis/pipelines/1/runs/14/signedlogcontent/3?urlExpires=2020-05-17T14%3A57%3A49.7161030Z&amp;urlSigningMethod=HMACV1&amp;urlSignature=mRC%2BFoU1ZvMWi6APsmFTtD%2Btwdj9GIlkPMeb6rX5Hhw%3D" rel="nofollow noopener">https://pipelines.actions.githubusercontent.com/rJZXGyMx0DPiJ1uuZOVUzJH5mFZ5flbMm8uZtBjQquT4PzUNKW/_apis/pipelines/1/runs/14/signedlogcontent/3?urlExpires=2020-05-17T14%3A57%3A49.7161030Z&amp;urlSigningMethod=HMACV1&amp;urlSignature=mRC%2BFoU1ZvMWi6APsmFTtD%2Btwdj9GIlkPMeb6rX5Hhw%3D</a>) and observe that another message actually exists:</p>
<pre><code>2020-05-17T14:56:16.7680142Z run-clang-format.py: error: unrecognized arguments: [...]  
</code></pre>
<p>Having such messages swalloed by GHA makes itz extremly unreliable as usually people rely on the parsed output and rarely look at the raw output</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/flamefire">@flamefire</a> ,</p>
<p>I notice the error is displayed on the action log, could you please help to confirm?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/94538cbe53d2473fcd4f91167c9d29176d24a3ef.png" data-download-href="/uploads/short-url/la9Fyg5dal3bQu8DDWj09dW7Nh5.png?dl=1" title="error.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181100925-cca05036-cfd3-4596-83b9-f73e1ba74c75.png" alt="error.png" data-base62-sha1="la9Fyg5dal3bQu8DDWj09dW7Nh5" width="690" height="117" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/94538cbe53d2473fcd4f91167c9d29176d24a3ef_2_690x117.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/94538cbe53d2473fcd4f91167c9d29176d24a3ef_2_1035x175.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/9/94538cbe53d2473fcd4f91167c9d29176d24a3ef_2_1380x234.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/94538cbe53d2473fcd4f91167c9d29176d24a3ef_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">error.png</span><span class="informations">1518×258 31.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If the workflow logs do not provide enough detail to diagnose why a workflow, job, or step is not working as expected, you can enable additional debug logging, please check the official <a href="https://help.github.com/en/actions/configuring-and-managing-workflows/managing-a-workflow-run#enabling-debug-logging" rel="nofollow noopener">doc</a> here for more details.</p>
<p>Thanks.</p>

**Comment by Flamefire** [comment]
<p>Yes now it is there. But I can swear it wasn’t there when I reported this. Will take a screenshot next time…</p>
<p>Maybe it was fixed in the meantime or it was a random fluke. Anyway, thanks!</p>
